docs/docs-ref-autogen/toc.yml

```diff
@@ -540,6 +544,78 @@ items:
                 uid: outlook.Office.Time
               - name: UserProfile
                 uid: outlook.Office.UserProfile
+          - name: Outlook 1.7
+            uid: Outlook_1_7
+            items:
+              - name: MailboxEnums
+                items:
+                  - name: AttachmentType
+                    uid: Outlook_1_7.Office.MailboxEnums.AttachmentType
+                  - name: EntityType
+                    uid: Outlook_1_7.Office.MailboxEnums.EntityType
+                  - name: ItemNotificationMessageType
+                    uid: >-
```

Comment:
Double check this is showing up correctly.

Reply:
No issue here.